kernel/kernel-api/src/main/java/io/delta/kernel/internal/metrics/SnapshotQueryContext.java

```diff
@@ -99,12 +108,15 @@ public SnapshotMetrics getSnapshotMetrics() {
   }
 
   public String getQueryDisplayStr() {
```

Comment:
Since even AS OF LATEST queryies eventually set the version, this was incorrectly returning AS OF VERSION X.

Now, it will return `LATEST SNAPSHOT (RESOLVED TO VERSION X)` which is much clearer.